jwst/regtest/test_nirspec.py

```diff
@@ -0,0 +1,46 @@
+"""Regression tests for NIRSpec"""
+import os
+
+import pytest
+
+from . import regtestdata as rt
+
+
+@pytest.fixture(scope='module')
+def run_nrs_ifu(jail, rtdata_module):
+    """Run the pipelines"""
+    step_params = {
+        'input_path': 'nirspec/spectroscopic/nrs_ifu_uncal.fits',
+        'step': 'calwebb_detector1.cfg',
```

Comment:
The request contained in JP-1199 was to test the ``calwebb_spec2`` pipeline on NIRSpec IFU data, not the ``calwebb_detector1`` pipeline.

Reply:
At the time, did not see any nirspec-related detector1 tests, so was going to cover both. However, if it is covered, I can pre-build the Stage 1 files.

Reply:
We have ``regtest/test_nirspec_subarray.py``, which runs ``calwebb_detector1`` on a NIRSpec subarray exposure. So we do have at least one detector1 test on NIRSpec data, although it's subarray (intentionally so, in order to test the ``gain_scale`` step) and https://jira.stsci.edu/browse/JP-1169 is in progress to create a detector1 test on a full-frame NIRSpec exposure using the IRS2 readout pattern. So once the latter gets finished, we should have decent coverage for NIRSpec detector1.